promethion_transfer.py

```diff
@@ -0,0 +1,76 @@
+""" Transfers new PromethION runs to ngi-nas using rsync.
```

Comment:
Should this be a part of TACA? The Illumina runs are transferred with lsync, so I'm not quite sure where this should go.

Reply:
I think the plan was first to have it as a cronjob but we needed some additional logic (keeping track of which sync is the last) so it made sense to make it a python script. I think TACA is a good place to keep it yes

Reply:
Sure, I can move it to TACA instead 👍

Reply:
Oh, no I'm sorry! I thought this PR already was on TACA... I remember now that we discussed that we shouldn't install more than absolutely necessary on the PromethION. Again, sorry for the mess and all the extra work